app/admin/views.py

```diff
@@ -70,12 +102,38 @@ def change_user_email(user_id):
     if form.validate_on_submit():
         user.email = form.email.data
         db.session.add(user)
+        db.session.commit()
         flash('Email for user {} successfully changed to {}.'
               .format(user.full_name(), user.email),
               'form-success')
     return render_template('admin/manage_user.html', user=user, form=form)
 
 
+@admin.route('/user/<int:user_id>/change-account-type',
+             methods=['GET', 'POST'])
+@login_required
+@admin_required
+def change_account_type(user_id):
+    """Change a user's account type."""
+    if current_user.id == user_id:
+        flash('You cannot change the type of your own account. Please ask '
+              'another administrator to do this.', 'error')
+        return redirect(url_for('admin.user_info', user_id=user_id))
+
+    user = User.query.filter_by(id=user_id).first()
```

Comment:
I think `.get(id)` would also be better here. Probably the same actual implementation, but there may be some schnazzy stuff in the background if it's primary key.